src/hooks/useFetch.js

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect } from 'react';
+
+// 1. 우선 fetch 기본 함수 제작 후
+// 2. { get put patch delete post } Hooks를 추가적으로 제작
+// 3. /{team}/messages/{id}/, /{team}/recipients/{id}/, /{team}/recipients/{recipient_id}/reactions/
+// 4. url 가공 필요
+
+// 기본 fetch에서 url, method, body을 받아 실행
+const baseURL = 'https://rolling-api.vercel.app';
+const useFetch = (url, method = 'GET', body) => {
+  const [isData, setIsData] = useState(null);
```

Comment:
보통 변수 앞에 is나 has가 붙으면 `boolean` 값이 들어가 있다고 생각할 수 있어서 is는 지우는 것이 더 나을 거 같습니다
```suggestion
  const [data, setData] = useState(null);
```